src/traverse.js

```diff
@@ -9,7 +9,7 @@ var sequence = require('./sequence');
  * then uses [`sequence`](#sequence) to transform the resulting Traversable of Applicative
  * into an Applicative of Traversable.
  *
- * Dispatches to the `sequence` method of the third argument, if present.
+ * Dispatches to the `traverse` method of the third argument, if present.
```

Comment:
The docs state that the function supports Fantasy Land traversables. I don't think it's necessary to mention that it dispatches to `fantasy-land/traverse`. Can't we consider that to be more of an implementation detail? Users should just have to think about the abstractions.